okhttp/src/main/java/okhttp3/EventListener.java

```diff
@@ -87,15 +87,15 @@ public void callStart(Call call) {
    * <p>If the {@link Call} is able to reuse an existing pooled connection, this method will not be
    * invoked. See {@link ConnectionPool}.
    */
-  public void dnsStart(Call call, String domainName) {
+  public void dnsStart(@Nullable Call call, String domainName) {
```

Comment:
Why are these now Nullable?

Reply:
1. There is a package-level *ParametersAreNonnullByDefault* for that class' package, so, *null*-checks are generated for all method parameters not marked by *Nullable*
2. `mvn clean package` shows that *null* is given there (at least in tests), hence, we get an *NPE*

Reply:
We should fix those tests then